printer_mods/Takuya/USBCameraMount/ReadMe.md

```diff
@@ -0,0 +1,21 @@
+This my my contribution to the awesome Voron Community!
+
+Its a camera mount for a USB Camera.
+
+![Screenshot_4](https://user-images.githubusercontent.com/54855101/126408342-3fb0ea8e-ef64-45e4-821c-6806f4884df7.jpg)
+
+
+Items needed:
+2 M3x8 Bolts
```

Comment:
Please make it a markdown list, otherwise it appears all on one line.

<img width="479" alt="Screen Shot 2021-08-04 at 17 50 18" src="https://user-images.githubusercontent.com/69389593/128259817-28743bbd-2620-4861-87b4-3c01b7d0ad4e.png">

Reply:
Thanks,
done